static/app/components/u2f/u2finterface.tsx

```diff
@@ -1,18 +1,22 @@
 import * as React from 'react';
 import * as Sentry from '@sentry/react';
+// @ts-ignore
+import * as cbor from 'cbor-web';
```

Comment:
There's also https://github.com/jprochazk/cbor ?

Reply:
might be worth switching out, looks like a smaller footprint size wise and slightly more ergonomic apis.

Reply:
Just tried using this, it wasn't able to decode the byte array i passed in